charts/cmc-claim-store/values.aat.template.yaml

```diff
@@ -5,8 +5,6 @@ java:
     PDF_SERVICE_URL: http://cmc-pdf-service-java.software-engineering
     SEND_LETTER_URL: http://rpe-send-letter-service-java.software-engineering
     DOCUMENT_MANAGEMENT_URL: http://dm-store-java.ccd
-    PAY_URL: false
-    FEES_URL: false
```

Comment:
Please don't remove these.

Reply:
don't worry about it ... it's already in values.yaml file which is the default values